includes/validate.php

```diff
@@ -362,12 +362,12 @@ function edac_get_content( $post ) {
 				}
 			}
 		} catch ( Exception $e ) {
-			update_post_meta( $post->ID, '_edac_density_data', '0,0' );
+			update_post_meta( $post->ID, '_edac_density_data', array( 0, 0 ) );
```

Comment:
I'm curious why Matt had this set as a string rather than an array and if that was intentional. I did a quick test with the Audit History add-on and I didn't notice any issues. Do you thing it needs further investigation?

Reply:
I dug back through past versions of the files that set and use this and it seems as though it's been expected to be an array with 2 keys going back a very long way. 

I suspect that this was just added as a catch-all default value and wasn't easy to trigger due to this code only executing under uncertain error conditions that are hard to replicate. This being a string here seems like just an assumption that couldn't be tested under normal operations. It would always have caused issues if it stored this to be read later on.

Reply:
This issue is the last time this code was edited but even back then it expected that the value would be an array returned with this array and 2 keys inside it. https://github.com/equalizedigital/accessibility-checker/issues/245